Packs/PrismaCloud/ReleaseNotes/4_2_23.json

```diff
@@ -0,0 +1,4 @@
+{
+    "breakingChanges": true,
+    "breakingChangesNotes": "The output now lacks the folowing 2 values by default: ***data.items.data.disks.shieldedInstanceInitialState***, ***data.items.data.metadata.items.configure-sh***. Use the argument ****include_additional_resource_fields=true**** to include their values in the output."
```

Comment:
Explain here, if a user needs one those values, what they should do

Reply:
It's there:
 Use the argument ****include_additional_resource_fields=true**** to include their values in the output.